### DISCUSSION TITLE
v4.2.0

### DISCUSSION THREAD

**Discussion by klaasnicolaas** [klaasnicolaas/home-assistant-glow#584] [Announcements]
## What's changed

Check also the release blog: https://glow-energy.io/blog/release-4.2.0

_To receive a notification on new releases, click on **Watch** > **Custom** > **Releases** on the top._

## ✨ New features

- #582 Switch to version 3 UI of web server @klaasnicolaas

## 🐛 Bug fixes

- #562 Fix example in FAQ about using method @klaasnicolaas

## 🧰 Maintenance

- #549 Fix small issue in CI @klaasnicolaas

## ⬆️ Dependency updates

<details>
<summary>26 changes</summary>

- #550 ⬆️ Update dependency @easyops-cn/docusaurus-search-local to v0.44.4 @renovate
- #551 ⬆️ Lock file maintenance @renovate
- #552 ⬆️ Update dependency typescript to v5.5.4 @renovate
- #554 ⬆️ Lock file maintenance @renovate
- #555 ⬆️ Update esphome/build-action action to v3.3.0 @renovate
- #558 ⬆️ Update actions/upload-artifact action to v4.3.5 @renovate
- #560 ⬆️ Lock file maintenance @renovate
- #563 ⬆️ Update actions/upload-artifact action to v4.3.6 @renovate
- #567 ⬆️ Update dependency @docusaurus/tsconfig to v3.5.1 @renovate
- #568 ⬆️ Update dependency @easyops-cn/docusaurus-search-local to v0.44.5 @renovate
- #569 ⬆️ Update dependency @docusaurus/types to v3.5.1 @renovate
- #565 ⬆️ Update dependency @docusaurus/module-type-aliases to v3.5.1 @renovate
- #566 ⬆️ Update dependency @docusaurus/preset-classic to v3.5.1 @renovate
- #564 ⬆️ Update dependency @docusaurus/core to v3.5.1 @renovate
- #571 ⬆️ Update dependency @docusaurus/module-type-aliases to v3.5.2 @renovate
- #573 ⬆️ Update dependency @docusaurus/tsconfig to v3.5.2 @renovate
- #572 ⬆️ Update dependency @docusaurus/preset-classic to v3.5.2 @renovate
- #570 ⬆️ Update dependency @docusaurus/core to v3.5.2 @renovate
- #574 ⬆️ Update dependency @docusaurus/types to v3.5.2 @renovate
- #575 ⬆️ Lock file maintenance @renovate
- #577 ⬆️ Lock file maintenance @renovate
- #559 ⬆️ Update esphome/build-action action to v4 @renovate
- #578 ⬆️ Update klaasnicolaas/action-pr-labels action to v2.0.1 @renovate
- #579 ⬆️ Update dependency prism-react-renderer to v2.4.0 @renovate
- #580 ⬆️ Update actions/upload-artifact action to v4.4.0 @renovate
- #581 ⬆️ Lock file maintenance @renovate
</details>

**Full Changelog**: https://github.com/klaasnicolaas/home-assistant-glow/compare/4.1.1...4.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/klaasnicolaas/home-assistant-glow/releases/tag/4.2.0'>v4.2.0</a>.</em>

**Comment by s9550110** [comment]
I couldn't get this working, I've reverted back to 3.0

**Reply by klaasnicolaas** [reply]
Try to use v4.2.1 there were some problems with v4.2.0. If you continue to experience problems please create an issue with additional information.

./Klaas
